test/tests/functional/pbs_Rrecord_resources_used.py

```diff
@@ -404,7 +404,7 @@ def test_t5(self):
             '.*Exit_status=-11.*.*resources_used.*.*run_count=1.*',
             id=jid3s1, regexp=True)
 
-    def test_t6(self):
+    def test_Rrecord_with_select_and_multiple_rerun_of_job(self):
```

Comment:
please make this name little short "test_Rrecord_with_multiple_reruns_select" or "test_Rrecord_with_multiple_reruns_case2"